pom.xml

```diff
@@ -56,21 +56,26 @@
 
   <properties>
     <vertx.version>3.8.3</vertx.version>
+    <rxjava.version>2.2.14</rxjava.version>
     <slf4j-api.version>1.7.29</slf4j-api.version>
+    <cloudevent.version>0.2.1</cloudevent.version>
+    <axle.version>0.0.10</axle.version>
+
+    <cdi.version>2.0.SP1</cdi.version>
     <microprofile-reactive-messaging.version>1.0</microprofile-reactive-messaging.version>
     <microprofile-reactive-streams.version>1.0.1</microprofile-reactive-streams.version>
-    <rxjava.version>2.2.14</rxjava.version>
+    <microprofile-config.version>1.3</microprofile-config.version>
+
     <smallrye-reactive-streams-ops.version>1.0.10</smallrye-reactive-streams-ops.version>
     <smallrye-config.version>1.4.1</smallrye-config.version>
-    <cloudevent.version>0.2.1</cloudevent.version>
-    <axle.version>0.0.10</axle.version>
 
     <asciidoctorj.version>1.5.8.1</asciidoctorj.version>
 
     <sonar.coverage.jacoco.xmlReportPaths>target/site/jacoco/jacoco.xml</sonar.coverage.jacoco.xmlReportPaths>
   </properties>
 
   <modules>
+    <module>reactive-messaging-api</module>
```

Comment:
should it be named smallrye-reactive-messaging-api?

Reply:
the artifact is named that, I just cheated on the directory name to have it at the top.

Can add "smallrye" for consistency if desired